config/config_test.go

```diff
@@ -0,0 +1,102 @@
+// Package config is responsible for loading user-manager application config.
+// Basic configuration like consul credentials and address, http port to listen for requests,
+// postgres schema name, credentials, and client timeout are read from environment variables.
+package config
+
+import (
+	"reflect"
```

Comment:
I would suggest to use assert in tests instead of reflect.

Reply:
Ok, used it

Reply:
But then do not need it